### PR TITLE
Adding extra arguments for Slack Message like reply to thread & markup

### PR DESCRIPTION
**Pull request by muneer22**
Fixes https://github.com/scrapinghub/spidermon/issues/293

- Initial push as WIP.
- I have some more queries regarding this.

### CODE REVIEW

File: spidermon/contrib/actions/slack/__init__.py [hunk 1]
```diff
@@ -110,14 +116,22 @@ def _send_user_message(
...
             attachments=self._parse_attachments(attachments),
             username=self.sender_name,
             icon_url=self._get_icon_url(),
+            thread_ts=self.thread_id,
+            # thread_ts = "1659182317.958909",
+            reply_broadcast=self.also_send_channel,
+            mrkdwn=self.mrkdwn
+        )
+
+        self.thread_id = (
```
Comment (curita) [inline spidermon/contrib/actions/slack/__init__.py:133]: **issue:** if we reuse the `thread_id` all the Slack messages will be grouped under the same thread. This is a nice feature to have in certain projects, but it should probably be implemented separately and not as the default behavior, or it'd break backward compatibility.

Comment (rennerocha) [reply spidermon/contrib/actions/slack/__init__.py:133]: The main reason of this issue was exactly to make it possible to send messages to the same thread (using `thread_ts` parameter). However, if we don't provide a thread_id, we shouldn't reuse the same for all messages.

Comment (curita) [reply spidermon/contrib/actions/slack/__init__.py:133]: You are right, should have read the ticket description 😅 Makes sense to rely on the provided `thread_id`, and leave it to users to define it.

**Comment by Gallaecio** [comment]
See also https://github.com/scrapinghub/spidermon/pull/346

**Comment by VMRuiz** [comment]
Closed in favor of more generic solution  https://github.com/scrapinghub/spidermon/pull/397/files

**Pull request closed without merging**